syntax = "proto3";
package basic;




import "tool_sdk/model/tool/tool_input.proto";
import "google/protobuf/struct.proto";

/*
CreateTool返回
*/
message CreateToolResponse {
    /*
    工具Id，服务端自动生成
    */
    string toolId = 1 ;
    /*
    工具版本
    */
    string vId = 2 ;
    /*
    禁用工具开关
    */
    bool disable = 3 ;
}

/*
CreateToolApi返回
*/
message CreateToolResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    basic.CreateToolResponse data = 4 ;
}

